test/extended/cmd.sh

```diff
@@ -143,3 +144,15 @@ os::cmd::expect_success "oc process -f test/extended/fixtures/image-pull-secrets
 os::cmd::try_until_text "oc get pods/my-dc-1-hook-pre -o jsonpath='{.status.containerStatuses[0].imageID}'" "docker"
 os::cmd::expect_success "oc delete all --all"
 os::cmd::expect_success "docker rmi -f ${docker_registry}/image-ns/busybox:latest"
+
+# check to make sure that service serving cert signing works correctly
```

Comment:
> I don't want to do a build in the tests to test something that doesn't need
> builds fundamentally (because they are much slower and have a bigger system
> impact).  I'll relax the extended test but please fix this.

It seems that it was convenient because there's not another sane or easy way to do it.  Upstream couldn't make nginx work as-is, so their examples point to a fork of the dockerhub nginx image that was built by someone else for his example.  I could maybe use that if you wanted.  Otherwise, this is as good as it gets for the moment.

Also, did you know that our default SCCs don't let someone run nginx anyway?